pset-functions.js

```diff
@@ -135,6 +176,47 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function scrabbleScore(word) {
+  // Have to split the sentence into words
+  let characters=word.split("")
+  let points=0
+  // put the cases inside arrays to facilitate adding points
+  const onePoint=["a","e","i","o","u","l","n","r","s","t"]
+  const twoPoint=["d","g"]
+  const threePoint=["b","c","m","p"]
+  const fourPoint=["f","h","v","w","y"]
+  const fivePoint=["k"]
+  const eightPoint=["j","x"]
+  const tenPoint=["q","z"]
+  //loop through the number of characters in the workd
+
+  for (var i = 0; i < characters.length; i++) {
+      // check which point categories the character belongs in and add points
+    if (onePoint.includes(characters[i])) {
+        points=points+1
+    } else if (twoPoint.includes(characters[i])) {
+        points=points+2
+    } else if (threePoint.includes(characters[i])) {
+      points=points+3
+    } else if (fourPoint.includes(characters[i])) {
+      points=points+4
+    } else if (fivePoint.includes(characters[i])) {
+      points=points+5
+    } else if (eightPoint.includes(characters[i])) {
+      points=points+8
+    } else if (tenPoint.includes(characters[i])) {
+      points=points+10
+    }//find in the point categories
+  } //for each charater in the test word
+  return points
+} // end of function
+
+const word="function"
+console.log("Test: " + word)
+console.log("Number of points: " + scrabbleScore(word))
```

Comment:
@AtleLohrmann Great work here on this problem. good use of the array method `includes` to check if each character against the point arrays and add the points up.
Take a look at this solution [here]('https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174')